internal/app/export_test.go

```diff
@@ -11,6 +11,10 @@ type MockAuthorizer struct {
 	RejectAuth bool
 }
 
+type MockProxy struct {
```

Comment:
I know these types and functions are only used in the tests, but we still would benefit a lot from having them documented (thinking about future maintenance)

Ps. This comment applies to the whole file, otherwise I would be spamming too much :sweat_smile: